src/core_plugins/table_vis/public/__tests__/_table_vis_controller.js

```diff
@@ -18,13 +18,13 @@
  */
 
 import $ from 'jquery';
-import _ from 'lodash';
 import expect from 'expect.js';
 import ngMock from 'ng_mock';
 import { LegacyResponseHandlerProvider } from 'ui/vis/response_handlers/legacy';
 import { VisProvider } from 'ui/vis';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
 import { AppStateProvider } from 'ui/state_management/app_state';
+import { tabifyAggResponse } from '../../../../ui/public/agg_response/tabify';
```

Comment:
in tests we used elasticsearch stubbed responses before, now we need to tabify them first.
this is far from ideal, but should be ok short term solution, as we plan to migrate our tests to the pipeline as soon as that is available.

Reply:
can't we write this import as `import { tabifyAggResponse } from 'ui/agg_response/tabify';`?